cli/package.json

```diff
@@ -14,7 +14,7 @@
     "build": "tsc -p tsconfig.build.json",
     "postbuild": "cp package.json ./build",
     "types": "tsc -p tsconfig.json --noEmit",
-    "test": "nyc ts-mocha --exit tests/**/*.test.ts",
+    "test": "nyc ts-mocha --exit tests/e2e/*.test.ts tests/unit/*.test.ts",
```

Comment:
Why we only test these tests in these folders? Is that because the folder **stress** is only run for nightly?

Reply:
correct, it would take too much time to run on every pr/commit